apps/docs/pages/guides/auth/auth-identity-linking.mdx

```diff
@@ -0,0 +1,71 @@
+| updated_at      | `string` | The timestamp that the identity was last updated.                                                                                                                                                                                     |
+
+## Identity linking strategies
+
+Currently, Supabase Auth supports 2 strategies to link an identity to a user:
+
+1. [Automatic Linking](#automatic-linking)
+2. [Manual Linking](#manual-linking-beta)
+
+### Automatic linking
+
+Supabase Auth automatically links identities with the same email address to a single user. This helps to improve the user experience when multiple OAuth login options are presented since the user does not need to remember which OAuth account they used to sign up with. When a new user signs in with OAuth, Supabase Auth will attempt to look for an existing user that uses the same email address. If a match is found, the new identity is linked to the user.
+
+In order for automatic linking to correctly identify the user for linking, Supabase Auth needs to ensure that all user emails are unique. It would also be an insecure practice to automatically link an identity to a user with an unverified email address since that could lead to pre-account takeover attacks. To prevent this from happening, when a new identity can be linked to an existing user, Supabase Auth will remove any other unconfirmed identities linked to an existing user.
+
+Users that signed up with [SAML SSO](/docs/guides/auth/sso/auth-sso-saml) will not be considered as targets for automatic linking.
+
+### Manual linking (Beta)
+
+Supabase Auth allows a user to initiate identity linking with a different email address when they are logged in. To link an OAuth identity to the user:
+
+```js
+const { data, error } = await supabase.auth.linkIdentity({ provider: 'google' })
+```
+
+In the example above, the user will be redirected to Google to complete the OAuth2.0 flow. Once the OAuth2.0 flow has completed successfully, the user will be redirected back to the application and the google identity will be linked to the user. You can enable manual linking from your project's authentication [configuration options](/dashboard/project/_/settings/auth).
+
+## Unlink an identity
+
+To unlink an identity from a user, the user needs to be logged in first and have at least 2 linked identities.
+
+```js
+// retrieve all identities linked to a user
+const {
+  data: { identities },
+} = await supabase.auth.getUserIdentities()
+
+// find the google identity linked to the user
+const googleIdentity = identities.find((identity) => identity.provider === 'google')
+
+// unlink the google identity from the user
+const { data, error } = await supabase.auth.unlinkIdentity(googleIdentity)
+```
```

Comment:
Might be useful to link directly to the reference docs on those methods somewhere.

Reply:
reference docs are not updated yet because the client lib PR hasn't been merged